doc/building_macos.md

```diff
@@ -0,0 +1,102 @@
+# Building Julius for Mac OS
+
+## Installing the dependencies
+
+In order to be able to build Julius, you need to install the dependencies first.
+
+The dependencies are:
+
+* The command line developer tools;
+* Homebrew;
+* CMake;
+* The `SDL` library;
+* The `SDL_mixer` library.
+
+If you already have all of them installed, you can proceed to [build Julius](#building-julius).
+
+
+### Installing the command line developer tools
+
+To install the command line developer tools, open a terminal window and type:
+
+    $ xcode-select --install
+
+The following window should appear:
+
+![](images/building_macos_1.png)
```

Comment:
Do we really need screenshots here? I'd like to keep the number of images in the source to a minimum.

Reply:
Well was tailoring people with little programming experience and I don't know much about mac, so I leave that option open to you. I'll remove the image then.